share/spice/pubmed/pubmed.js

```diff
@@ -0,0 +1,97 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_pubmed = function(api_result){
+        if (!api_result || api_result.error) {
+            return Spice.failed('pubmed');
+        }
+        api_result = api_result.esearchresult.idlist;
+        var url = 'http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi?db=pubmed&retmode=json&rettype=abstract&retmax=10&id=';
```

Comment:
@b2ddg this is considered a privacy leak, because the client makes a direct request to the 3rd party.

The solution is to create a "dummy" spice, `DDG::Spice::Pubmed::Data` with a `spice to`, but no callback (or `wrap_jsonp`)

Then you hit the nginx endpoint for that package from here:

```js
$.getJSON("/js/spice/pubmed_data/foo/bar", function(data){
  item.set("description", data.description);
}
```

Make sense?

XKCD does this (and so do a few other IAs): https://github.com/duckduckgo/zeroclickinfo-spice/search?utf8=%E2%9C%93&q=getJSON